script/start_server.sh.sample

```diff
@@ -50,6 +50,19 @@ if ! command -v $PUMA_CMD 2>&1 >/dev/null; then
   exit 1
 fi
 
+# Pick a system stack:
+# UNCOMMENT AS REQUIRED ON NON SAMPLE FILE
```

Comment:
Capistrano deployment should copy this file in place over the existing scripts `.sample` is now really a bit of a misleading extension...

Reply:
(just as an "FYI")